tests/main/snap-set/task.yaml

```diff
@@ -1,21 +1,20 @@
 summary: Check that `snap set` runs configure hook.
 
 prepare: |
+    #shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB"/snaps.sh
+
     echo "Build basic test package (without hooks)"
-    snap pack $TESTSLIB/snaps/basic
-    snap install --dangerous basic_1.0_all.snap
+    install_local basic
 
     echo "Build failing hooks package"
     snap pack $TESTSLIB/snaps/failing-config-hooks
 
     echo "Build package with hook to run snapctl set"
-    snap pack $TESTSLIB/snaps/snapctl-hooks
-    snap install --dangerous snapctl-hooks_1.0_all.snap
+    install_local snapctl-hooks
 
 restore: |
-    rm -f basic_1.0_all.snap
     rm -f failing-config-hooks_1.0_all.snap
```

Comment:
@zyga, in this case yes, because we need to clean up the snap failing-config-hooks, because we are packing it and then installing it manually during the test execution.